shaders/src/getters.vs

```diff
@@ -142,12 +142,14 @@ vec4 computeWorldPosition() {
     mat4 transform = getWorldFromViewMatrix();
     vec3 position = getPosition().xyz;
     return mulMat4x4Float3(transform, position);
-#else
+#elif defined(VERTEX_DOMAIN_DEVICE)
     mat4 transform = getWorldFromClipMatrix();
     vec4 position = transform * getPosition();
     if (abs(position.w) < MEDIUMP_FLT_MIN) {
         position.w = position.w < 0.0 ? -MEDIUMP_FLT_MIN : MEDIUMP_FLT_MIN;
     }
     return position * (1.0 / position.w);
+#else
+#error "Unknown Vertex Domain"
```

Comment:
```suggestion
#error Unknown Vertex Domain
```

IIRC Chrome can't handle quotes inside shader text. I think these are technically disallowed characters.

Reply:
Also is it actually needed? These defines are generated by filamat. It shouldn't ever get to the #else case.

Reply:
Well, unless we add a constant one day and have a bug or something.